lib/fastlane/plugin/synx/actions/synx_action.rb

```diff
@@ -12,7 +12,10 @@ def self.run(params)
         cmd << ["--no-default-exclusions"] if params[:no_default_exclusions]
         cmd << ["--no-sort-by-name"] if params[:no_sort_by_name]
         cmd << ["--quiet"] if params[:quiet]
-        cmd << ["--exclusion #{params[:exclusion]}"] if params[:exclusion]
+        if params[:exclusion]
+          Array(params[:exclusion]).each {|exclusion|
+            cmd << ["--exclusion #{exclusion}"] }
```

Comment:
@afonsograca Thanks for review!
I think that it would be better to use [Shellwords.join()](http://ruby-doc.org/stdlib-2.0.0/libdoc/shellwords/rdoc/Shellwords.html#method-c-join).

_Example_

```ruby
Shellwords.join([
  "synx",
  "-p",
  "--exclusion",
  "/OCMock/Core Mocks",
  "Source/OCMock.xcodeproj/"])
#=> synx -p --exclusion /OCMock/Core\ Mocks Source/OCMock.xcodeproj/
```